src/node_crypto.cc

```diff
@@ -4841,7 +4841,7 @@ void PBKDF2(const FunctionCallbackInfo<Value>& args) {
                           pass,
                           saltlen,
                           salt,
-                          iter,
+                          static_cast<ssize_t>(iter),
```

Comment:
I believe this has the same error as what we have recently fixed. Please take a look at da3f42550. This code needs `INT_MAX` check too.